pyfar/dsp/fft.py

```diff
@@ -292,7 +395,7 @@ def _is_odd(num):
     return bool(num & 0x1)
 
 
-def _n_bins(n_samples):
+def _calc_n_bins_from_time_data(n_samples, complex=False):
```

Comment:
```suggestion
def _n_bins_from_n_samples(n_samples, complex_time=False):
```
1. Please avoid overwriting the in-built complex dtype
2. The number of bins is  calculated from the number of samples, not the time data.